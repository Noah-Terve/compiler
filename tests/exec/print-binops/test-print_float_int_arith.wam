print @l float @r(4 + 4.9);
print @l float @r('c' + 5.9);