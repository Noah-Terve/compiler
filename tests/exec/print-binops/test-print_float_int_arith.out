8.9104.9